definitions/private-law/json/CaseEventToFields/SubmitAndPay/CaseEventToFields.json

```diff
@@ -187,12 +132,25 @@
     "LiveFrom": "02/12/2021",
     "CaseTypeID": "PRLAPPS",
     "CaseEventID": "submitAndPay",
-    "CaseFieldID": "paymentServiceRequestReferenceNumber",
+    "CaseFieldID": "helpWithFees",
     "PageFieldDisplayOrder": 1,
-    "DisplayContext": "READONLY",
+    "DisplayContext": "Mandatory",
+    "PageID": 3,
+    "ShowSummaryChangeOption": "Y",
+    "PageDisplayOrder": 3
+  },
+  {
+    "LiveFrom": "02/12/2021",
+    "CaseTypeID": "PRLAPPS",
+    "CaseEventID": "submitAndPay",
+    "CaseFieldID": "helpWithFeesNumber",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "Mandatory",
```

Comment:
you might want to use the existing case field for this so that it become visible in the case tab. we will have to look into it